common/src/main/java/no/difi/meldingsutveksling/HashBiMap.java

```diff
@@ -0,0 +1,29 @@
+package no.difi.meldingsutveksling;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HashBiMap<K, V> extends HashMap<K, V> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2160)